resource/labeling-app/components/text-sample/textArea.js

```diff
@@ -119,24 +142,28 @@ const TextArea = {
         resetSelection() {
             const textarea = document.getElementById('textarea');
```

Comment:
I must've missed it in the previous PRs, but `document.getElementById` means that you won't be able to have more than one TextArea widgets on the screen (even if it's not crucial for our use case)
Alternative to that would be using `this.$el.getElementsByClassName` or `ref='something'` in the template and then `this.$refs.something.getElementsByClassName` in the js

Reply:
Mmmh, is it a bad thing to say that there is only one textarea ?

Reply:
For now, I think let's keep it as it is, but since in the beginning it wouldn't have required any additional effort from us to make this widget universal (instead of imposing this constraint with a single instance) we should've done so. I'll create a chore card in case we get to it in the future

Reply:
Perfect, let's do that